src/plugins/xerces/README.md

```diff
@@ -2,10 +2,11 @@
 - infos/author = e1528532 <e1528532@libelektra.org>
 - infos/licence = BSD
 - infos/provides = storage/xml
+- infos/needs =
 - infos/placements = getstorage setstorage
 - infos/status = recommended experimental unittest
+- infos/metadata = elektraXercesOriginalRootName
```

Comment:
> Btw. github cuts away XML.

```xml
<that>
  <is>
    <not>
      <necessarily>
        <true>
          👋
        </true>
      </necessarily>
    </not>
  </is>
</that>
```
However, since Markdown supports [inline HTML](https://daringfireball.net/projects/markdown/syntax#html) I would assume the GitHub Markdown converter just throws away unknown tags inside Markdown files.